filebeat/input/udp/config.go

```diff
@@ -12,7 +14,7 @@ var defaultConfig = config{
 		Type: "udp",
 	},
 	Config: udp.Config{
-		MaxMessageSize: 10240,
+		MaxMessageSize: 1 * humanize.KiByte,
```

Comment:
Sorry for the late comment, but I just realised we are changing the default here from 10KB to 1KB?